src/core/request.js

```diff
@@ -44,13 +44,15 @@ Request.prototype.request = function(servers, callback_) {
   const callback = typeof servers === 'function' ? servers : callback_;
 
   this.emit('before');
+
+  const wasRequested = this.requested;
```

Comment:
I've made changes that throws error in case of request called twice
https://github.com/ripple/ripple-lib/pull/572